apf/apfComplex.h

```diff
@@ -0,0 +1,14 @@
+#ifndef APFCOMPLEX_H_
+#define APFCOMPLEX_H_
+
+#ifdef C_COMPLEX
+  #include <complex.h>
+#endif
+
+#define CXX_COMPLEX 1
```

Comment:
Oh yeah, need to add this as a CMake option rather that just hard defining it to be CXX_COMPLEX for now.